engine/runtime/src/test/java/org/enso/compiler/EnsoCompilerTest.java

```diff
@@ -407,6 +407,31 @@ public void testMultiParameterFunction() throws Exception {
     """);
   }
 
+  @Test
+  @Ignore // Documented type signatures not supported yet in TreeToIr.
```

Comment:
I have a feeling this test still fails. I see:
![obrazek](https://user-images.githubusercontent.com/26887752/196501901-ff8a4c5a-5c9a-4164-8c76-ab39e5725934.png)